generated/nidaqmx/error_codes.py

```diff
@@ -6,6 +6,8 @@
 
 
 class DAQmxErrors(IntEnum):
+    REMOTE_SENSE = -209888
+    OVER_TEMPERATURE_PROTECTION_ACTIVATED = -209887
```

Comment:
These two Error codes are added because of the https://github.com/ni/grpc-device-scrapigen/pull/136